ccd-definition/CaseTypeTab/DocumentsHearingWelsh-AHN-CUI-nonprod.json

```diff
@@ -0,0 +1,11 @@
+[
+  {
```

Comment:
I thought the initial idea was to have Translated document stored within hearingDocuments, but this approach might be better.